docs/src/examples.md

```diff
@@ -0,0 +1,42 @@
+# Examples
+
+Here you can find a list of examples to help you quick start using this
+library.
+
+
+# 1D signal
+
+A simple real ``60 Hz`` with the first harmonics:
+
+```julia
+using Plots
+using FFTW
+using DSP
+
+# Number of points
+N = 2^12 - 1
+# Sample spacing
+Ts = 1 / (1.1 * N)
+# Sample rate
+fs = 1 / Ts
```

Comment:
together, this is equivalent to saying that the sample rate is `1.1 * N` Hz, i.e. 4504.5